specification/relay/resource-manager/Microsoft.Relay/common/v1/definitions.json

```diff
@@ -86,6 +86,59 @@
           "type": "string"
         }
       }
+    },
+    "SystemData": {
```

Comment:
Make sure they are same as common types https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#L528 (ideally re-use that, but if not convenient, at least make sure they are same)

Reply:
changed to use from common type